### PR TITLE
fix: add jest coverage config to package.json

### PR DESCRIPTION
**Pull request by lynnshaoyu**
Configure package.json to show test coverage. Target coverage thresholds set to 70% currently. 

To run tests with coverage info: 
yarn test --coverage --watchAll

Run only tests affected by your changes:
yarn test --coverage --watch

Also added testing instructions to CONTRIBUTING.md in this PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
 1. `yarn install`
 1. `yarn build`
 
+## Running tests
+
+1. Run unit tests: `yarn test`
```
Comment (zzmp) [inline CONTRIBUTING.md:19]: These are all distinct, so they should be bullets, not numbered (`-`).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
...
+1. Run unit tests and show test coverage info for all tests: `yarn test --coverage --watchAll`
```
Comment (zzmp) [inline CONTRIBUTING.md:20]: I don't think this is necessary - it's more for a maintainer than a contributor.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
...
+1. Run unit tests and show test coverage info only for tests affected by your changes: `yarn test --coverage --watch`
```
Comment (zzmp) [inline CONTRIBUTING.md:21]: This should be the standard command. Is there any reason to not include `--coverage` in `yarn test`?
Also, `--watch` is done by default through jest, and does not need to be included.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
...
+1. Run e2e cypress tests with visual interface: `yarn cypress open`
```
Comment (zzmp) [inline CONTRIBUTING.md:22]: In order to run cypress tests, you'll need to a build to test them off of.
You'll need to `yarn build:e2e` in order to get a build with infura keys that will work on your localhost (vs `yarn build`, which is for production), and then `yarn test:e2e` will open cypress (it aliases `cypress open`) with a server backing the tests. CI uses `yarn test:e2e:ci` to run without the cypress UI.

File: package.json [hunk 5]
```diff
@@ -156,6 +156,27 @@
...
+      "src/lib/hooks/**/*.ts*", 
+      "src/lib/state/**/*.ts*", 
+      "src/lib/utils/**/*.ts*", 
+      "src/pages/**/*.ts*",
+      "src/state/**/*.ts*",
+      "src/utils/**/*.ts*"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 70,
```
Comment (zzmp) [inline package.json:173]: These may be too high, considering where we are currently at.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (lynnshaoyu) [reply CONTRIBUTING.md]: Done

File: package.json (hunk 5 above)
Comment (lynnshaoyu) [reply package.json]: True, changed to 25

File: CONTRIBUTING.md (hunk 4 above)
Comment (lynnshaoyu) [reply CONTRIBUTING.md]: Thanks! I added these instructions for running integration tests

File: CONTRIBUTING.md (hunk 2 above)
Comment (lynnshaoyu) [reply CONTRIBUTING.md]: I was thinking maybe adding these would encourage contributors to pay attention to / add their own unit tests when submitting PRs. The commands are pretty standard but for some newer contributors they might not be familiar with reading the scripts section of package.json and the coverage watchAll feature of jest.

File: package.json (hunk 5 above)
Comment (lynnshaoyu) [reply package.json]: I lowered them to just under where we are so our unit test check in the git automated checks don't block submissions.

File: package.json [hunk 6]
```diff
@@ -16,11 +16,32 @@
...
+      "src/hooks/**/*.ts*", 
+      "src/lib/hooks/**/*.ts*", 
+      "src/lib/state/**/*.ts*", 
+      "src/lib/utils/**/*.ts*", 
+      "src/pages/**/*.ts*",
+      "src/state/**/*.ts*",
+      "src/utils/**/*.ts*"
+    ],
+    "coverageThreshold": {
+      "global": {
```
Comment (zzmp) [inline package.json:37]: oof

**Review by zzmp — APPROVED** [review]

**Pull request merged**